odc/geo/geobox.py

```diff
@@ -237,7 +238,7 @@ def __and__(self, other) -> "GeoBox":
 
     def is_empty(self) -> bool:
         """Check if geobox is "empty"."""
-        return self._shape.shape == (0, 0)
+        return 0 in self._shape
```

Comment:
I think it will complicate type information without any real gain, in a statically typed language that might make sense, but in Python only people with mypy might "benefit" as you can type hint as "not empty geobox", but...